### ISSUE TITLE
Add more tests for radiation source model: radiation_profile

### ISSUE THREAD

**Issue by kj5248** [Fusion-Power-Plant-Framework/bluemira#3328]
Add more tests for radiation_profile.py. Current breakdown of functions within file that are tested (in test_radiation_profile):

Radiation
- ~~collect_flux_tubes~~
- ~~flux_tube_pol_t~~
- ~~flux_tube_pol_n~~
- mp_profile_plot

CoreRadiation
- ~~calculate_mp_radiation_profile~~
- plot_mp_radiation_profile
- calculate_core_distribution
- calculate_core_radiation_map
- radiation_distribution_plot
- plot_radiation_distribution
- plot_lz_vs_tref

ScrapeOffLayerRadiation
- collect_x_and_o_point_coordinates
- collect_separatrix_parameters
- ~~x_point_radiation_z_ext~~
- ~~radiation_region_ends~~
- radiation_region_points
- ~~mp_electron_density_temperature_profiles~~
- any_point_density_temperature_profiles
- ~~tar_electron_densitiy_temperature_profiles~~
- calculate_sector_distributions
- radiation_distribution_plot
- poloidal_distribution_plot
- plot_t_vs_n

DNScrapeOffLayerRadiation
- calculate_sol_distribution
- calculate_sol_radiation_distribution
- calculate_sol_radiation_map
- plot_poloidal_radiation_distribution

SNScrapeOffLayerRadiation
- calculate_sol_distribution
- calculate_sol_radiation_distribution
- calculate_sol_radiation_map
- plot_poloidal_radiation_distribution

RadiationSource
- ~~analyse~~
- ~~rad_core_by_psi_n~~
- ~~rad_core_by_points~~
- rad_sol_by_psi_n
- rad_sol_by_points
- rad_by_psi_n
- rad_by_points
- rad_map
- plot

**Comment by OceanNuclear** [comment]
After the PR #3800, all functions in RadiationSource has been tested.
However I don't have single-null tokamak data to be used for testing SNScrapeOffLayerRadiation. Perhaps @DarioV86 may be able to advise what to do about this?

**Comment by OceanNuclear** [comment]
Depends on #4256 getting merged, then the other two draft PRs can be converted to PR.

**Comment by OceanNuclear** [comment]
(Currently getting held up by @DarioV86 waiting to install WSL on his new laptop, this issue will be on hold until then)
